### ISSUE TITLE
linux install instructions don't include docker-compose instructions

### ISSUE THREAD

**Issue by nullvariable** [lando/docs#10]
Please complete the following:

**Link to the page that needs to be updated or changed.**
https://docs.lando.dev/basics/installation.html#linux

**Describe the change you'd like**
Requirements state that Docker CE is required, but don't note that docker compose is also required. When I followed these instructions on Ubuntu 20.10, I had to find and [install Docker Compose](https://docs.docker.com/compose/install/) separately. Would recommend that we add instructions for installing this as well. Particularly because Lando wants a specific version of it (I installed the latest and received an error message that it wasn't in the range lando preferred).

**Comment by reynoldsalec** [comment]
@labboy0276 does this match your Linux installation experiences?

**Comment by labboy0276** [comment]
I don't think we need docker-compose on Ubuntu.  I don't have it installed and Lando works for me:

```
john@tandem:~$ docker-compose --version

Command 'docker-compose' not found, but can be installed with:

sudo snap install docker          # version 20.10.8, or
sudo apt  install docker-compose  # version 1.25.0-1

See 'snap info docker' for additional versions.

john@tandem:~$ lando version
v3.6.0
```

**Comment by labboy0276** [comment]
Oddly enough I purged my system and needed to add in docker compose.

https://docs.docker.com/compose/install/
